docs/source/examples/replicate_alpaca-vicuna/ReplicateAlpacaVicuna.md

```diff
@@ -0,0 +1,146 @@
+    },
+```
+
+**The datasets that will be used in this tutorial are slightly modified versions of the original datasets.**
+They have been flattened into plain text files. Moreover all occurences of the “\n” symbol, which acts as example break in the OpenNMT world, have been replaced with '｟newline｠'.
+
+The onmt datasets can be retrieved at the links below:
+
+- [alpaca](https://opennmt-models.s3.amazonaws.com/llama/alpaca_clean.txt) (51751 examples)
+  
+- [vicuna](https://opennmt-models.s3.amazonaws.com/llama/sharegpt.txt) (28800 examples)
+
+Let us save the in a  local folder that we will name `dataAI`.
+
+Each example is a prompt that contains:
+- a short description of the task
+- an instrunction following the pattern `### Instruction`
+- a proposal of answer following the pattern `### Response` 
+
+Here is the first example in the onmt alpaca dataset:
+
+```txt
+Below is an instruction that describes a task. Write a response that appropriately completes the request.｟newline｠｟newline｠### Instruction:｟newline｠Give three tips for staying healthy.｟newline｠｟newline｠### Response:｟newline｠1. Eat a balanced and nutritious diet: Make sure your meals are inclusive of a variety of fruits and vegetables, lean protein, whole grains, and healthy fats. This helps to provide your body with the essential nutrients to function at its best and can help prevent chronic diseases.｟newline｠｟newline｠2. Engage in regular physical activity: Exercise is crucial for maintaining strong bones, muscles, and cardiovascular health. Aim for at least 150 minutes of moderate aerobic exercise or 75 minutes of vigorous exercise each week.｟newline｠｟newline｠3. Get enough sleep: Getting enough quality sleep is crucial for physical and mental well-being. It helps to regulate mood, improve cognitive function, and supports healthy growth and immune function. Aim for 7-9 hours of sleep each night.
+```
+
+## Finetuning
+
+We provide an example of a finetuning configuration (`replicate_alpaca-vicuna.yaml`). To enable the application of the LoRa method to the attention layers, the options of the checkpoint need to be overrided.
+
+
+The finetuning can be launched with this command:
+
+```shell 
+nohup python3 OpenNMT-py/onmt/bin/train.py -config replicate_alpaca-vicuna.yaml > finetenune-llama7B-vicuna-onmt.log &
+```
+
+We can start by generating some samples (by turning `dump_samples` to True and `n_samples` to a strictly positive value).
+
+It is worth noting that the he sentencepiece vocabulary does not map the custom substring ｟newline｠with specific token. However
+it maps the new line symbol '\n' with the token '<0x0A>'. To handle properlyour datasets without changing the vocabulary and training new embddings from scratch, the Tokenize transform replaces on-the-fly the token '｟newline｠' token with  '<0x0A>'.
```

Comment:
Few small typos and wording: `...with specific token.` -> `...with a specific token.`,  `properlyour` -> `properly our`, `embddings` -> `embeddings`